DESCRIPTION

```diff
@@ -261,9 +261,6 @@ Suggests:
 VignetteBuilder:
     knitr
 Remotes:
-    berndbischl/BBmisc,
```

Comment:
Was this change on purpose or bc the Travis build failed back then?

Reply:
Is there a reason why why install the devel version? If yes, we need to upload BBmisc before the next CRAN release. Otherwise it would be best to use the exact same version CRAN uses.

Reply:
AFAIK there was but probably all problems have been fixed? I released v2.14.0 ofc without the Remotes Field and everything seems to be fine. So yes, I guess we can drop these requirements.